FSJ_django20_project/FSJ/templates/FSJ/coord_student_list.html

```diff
@@ -0,0 +1,53 @@
+{% extends "FSJ/base_with_banner.html" %}
+{% load i18n %}
+
+{% block heading %}
+    Students
```

Comment:
Text should be translatable and include a comment for translators by using the {# #} and {% trans %} functions